filters/builtin/mod_header.go

```diff
@@ -0,0 +1,80 @@
+package builtin
+
+import (
+	"regexp"
+
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+)
+
+type modRequestHeader struct {
+	headerName  string
+	rx          *regexp.Regexp
+	replacement string
+	template    *eskip.Template
```

Comment:
This field doesn't seem to be used. Can you please delete it? We can improve the filter to use templates in a separate pass.

(`make staticcheck` detects the unused field, not sure now if this is only since we upgraded to go1.14)

Reply:
Removed.